instance/migrations/0031_merge.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
```

Comment:
It might be worthwhile to recreate the migrations on top of current master, to keep a linear list of migrations, which is simpler to manage.